ballerina/tests/bytes_io.bal

```diff
@@ -271,107 +190,63 @@ function testFileChannelReadBytesAsStream() returns Error? {
     string expectedString = "Sheldon Cooper";
     byte[] byteArr = [];
 
-    var fileOpenResult = openReadableFile(filePath);
-    if (fileOpenResult is ReadableByteChannel) {
-        var result = channelReadBlocksAsStream(fileOpenResult, 2);
-        if (result is stream<Block, Error?>) {
-            check result.forEach(function(Block val) {
-                                          foreach byte b in val {
-                                              byteArr.push(b);
-                                          }
-                                      });
-            string|error returnedString = langstring:fromBytes(byteArr);
-            if (returnedString is string) {
-                test:assertEquals(returnedString, expectedString);
-            } else {
-                test:assertFail(msg = returnedString.message());
-            }
-        } else {
-            test:assertFail(msg = result.message());
+    ReadableByteChannel fileOpenResult = check openReadableFile(filePath);
+    stream<Block, Error?> result = check channelReadBlocksAsStream(fileOpenResult, 2);
+    check result.forEach(function(Block val) {
+        foreach byte b in val {
+            byteArr.push(b);
         }
+    });
+    string|error returnedString = langstring:fromBytes(byteArr);
```

Comment:
```suggestion
    string returnedString = check langstring:fromBytes(byteArr);
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-io/pull/420/commits/917a8e482c0c914bc2ca9970fe31f935023fbdf4